src/modules/catalog.js

```diff
@@ -130,189 +131,35 @@ class Catalog {
   }
 
   /**
-   * Add item to index
+   * Adds multiple items to your index whilst updating existing ones (limit of 1,000)
    *
-   * @function addItem
+   * @function createOrReplaceItems
    * @param {object} parameters - Additional parameters for item details
-   * @param {string} parameters.item_name - The name of the item, as it will appear in the results
+   * @param {string} parameters.key - The API key of the index that you'd like to make changes to.
```

Comment:
Related to the other comment, I don't think we need key for these functions, it will come from `this.options`.